src/vstart.sh

```diff
@@ -246,7 +246,7 @@ options:
 	--no-parallel: dont start all OSDs in parallel
 	--jaeger: use jaegertracing for tracing
 	--seastore-devs: comma-separated list of blockdevs to use for seastore
-        --seastore-secondary-des: comma-separated list of secondary blockdevs to use for seastore
+        --seastore-secondary-des: comma-separated list of secondary blockdevs to use for seastore\n
```

Comment:
Why not put the `\n` on al line of its own.
Makes it less hidden/obscure, and easier to add a new option.

Reply:
Done. :)